client/src/featureform/register.py

```diff
@@ -2151,21 +2374,24 @@ def register_blob_store(
             description="An azure blob store provider to store offline and inference data"
         )
         ```
+
         Args:
-            name (str): Name of Azure blob store to be registered
-            container_name (str): Azure container name
-            root_path (str): custom path in container to store data
-            description (str): Description of Azure Blob provider to be registered
-            team (str): the name of the team registering the filestore
-            account_name (str): Azure account name
-            account_key (str): Secret azure account key
-            tags (List[str]): Optional grouping mechanism for resources
-            properties (dict): Optional grouping mechanism for resources
+            name (str): (Immutable) Name of Azure blob store to be registered
```

Comment:
out of scope but might be cool to have a way to easily identify mutable and immutable by an icon.